src/SonarScanner.MSBuild/Properties/AssemblyInfo.cs

```diff
@@ -19,7 +19,15 @@
  */
 
 using System.Reflection;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyTitle("SonarScanner.MSBuild")]
 [assembly: AssemblyProduct("SonarScanner.MSBuild")]
 [assembly: AssemblyDescription("")]
+
+// FIXME: add public key
```

Comment:
This needs to be addressed

Reply:
It belonged to a previous state of the base branch.
The issue was discussed [here](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1789#discussion_r1403203343) and fixed in [this commit](https://github.com/SonarSource/sonar-scanner-msbuild/pull/1789/commits/bf22a488aa01a4b601cd78948840b522f0da3afe) of that PR.

I have rebased again, removing all commits that belonged to #1789.
It should be OK now.